src/fastapi_poe/client.py

```diff
@@ -612,3 +612,13 @@ async def get_final_response(
     if not chunks:
         raise BotError(f"Bot {bot_name} sent no response")
     return "".join(chunks)
+
+
+def sync_bot_settings(
+    bot_name: str,
+    access_key: str = "",
+    base_url: str = "https://api.poe.com/bot/fetch_settings/",
+) -> None:
+    """Sync bot settings with the Poe server using bot name and its Access Key."""
```

Comment:
We can consider adding more context here to help the developer understand what this does.